src/atoms/TextComponents/textComponent.module.scss

```diff
@@ -0,0 +1,19 @@
+@import 'src/assets/stylesheets/global';
```

Comment:
so far we've been naming scss files `text-component.module.scss` etc - I don't mind either way but we should pick one and standardize